vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewHeaderAndFooterRowsIT.java

```diff
@@ -77,48 +68,43 @@ public void gridWithHeaderWithComponentRenderer_headerAndFooterAreRenderered() {
         scrollToElement(grid);
 
         GridTHTDElement headerCell = grid.getHeaderCell(0);
-        assertRendereredHeaderCell(headerCell, "<span>Name</span>", true, true);
+        assertRendereredHeaderCell(headerCell, "<span>Name</span>", true);
 
         headerCell = grid.getHeaderCell(1);
-        assertRendereredHeaderCell(headerCell, "<span>Age</span>", true, true);
+        assertRendereredHeaderCell(headerCell, "<span>Age</span>", true);
 
         headerCell = grid.getHeaderCell(2);
-        assertRendereredHeaderCell(headerCell, "<span>Street</span>", true,
-                false);
+        assertRendereredHeaderCell(headerCell, "<span>Street</span>", false);
 
         headerCell = grid.getHeaderCell(3);
-        assertRendereredHeaderCell(headerCell, "<span>Postal Code</span>", true,
+        assertRendereredHeaderCell(headerCell, "<span>Postal Code</span>",
                 false);
 
         Assert.assertTrue(
                 "There should be a cell with the renderered 'Basic Information' header",
-                hasComponentRendereredHeaderCell(grid,
-                        "<span>Basic Information</span>"));
+                hasComponentRendereredHeaderCell(grid, "span",
+                        "Basic Information"));
 
         Assert.assertTrue("There should be a cell with the renderered footer",
-                hasComponentRendereredHeaderCell(grid,
-                        "<span>Total: 500 people</span>"));
+                hasComponentRendereredHeaderCell(grid, "span",
+                        "Total: 500 people"));
     }
 
     private void assertRendereredHeaderCell(GridTHTDElement headerCell,
-            String text, boolean componentRenderer, boolean withSorter) {
+            String text, boolean withSorter) {
 
         String html = headerCell.getInnerHTML();
         if (withSorter) {
             Assert.assertTrue(html.contains("<vaadin-grid-sorter"));
         } else {
             Assert.assertFalse(html.contains("<vaadin-grid-sorter"));
         }
-        if (componentRenderer) {
-            Assert.assertTrue(html.contains("<flow-component-renderer"));
-        }
```

Comment:
Cells do not contain a `flow-component-renderer` tag anymore, remaining assertions should still verify that a component tag is rendered.